packages/nextjs/src/utils/nextjsOptions.ts

```diff
@@ -0,0 +1,11 @@
+import { NodeOptions } from '@sentry/node';
+import { BrowserOptions } from '@sentry/react';
+import { Options } from '@sentry/types';
+
+export interface NextjsOptions extends Options, BrowserOptions, NodeOptions {
+  /**
+   * A flag forcing the initialization of the SDK.
+   * By default, the SDK is only initialised in production.
+   */
+  forceInit?: boolean;
```

Comment:
I don't think that's the best choice for naming this option.
I would call it `dev: true` or `enableInDevelopment: true`

We use `forceInit` in our JS loader and it could be confused with that.